typescript/types/main.d.ts

```diff
@@ -90,6 +90,8 @@ type Widget = {
 };
 declare const WIDGETS: { [key: string]: Widget };
 
+type ShortBoolean = boolean | 0 | 1;
```

Comment:
Why not just use `true` or `false` if it's known beforehand, and cast to Boolean (by `Boolean(1)` or `!!1` or `1!=0`) if it's not?

Reply:
So here the intention is to enable this kind of usage, where anything truthy/falsy is expected:

```javascript
Bangle.setHRMPower(1, "myapp"); // equivalent to below - we tend to use 1 (or 0) instead of true/false
```

rather than:
```javascript
Bangle.setHRMPower(true, "myapp"); // longer to type, uses more storage, not the convention
```

And the advantage of typing this as `boolean | 0 | 1` means we don't open the door to any number being permitted, but we retain all the flexibility of javascript, with no runtime penalty (i.e. `Boolean(...)`, `!!1`, etc).

Reply:
Gordon probably agrees with you but IMO it's still better to write true, and I think tokenization might help with the memory problems.

Reply:
Agreed - the main rationale for the change is to support a long-term goal is to typecheck the existing JS, which makes use of this pattern

Reply:
Ah, good point